test/integration/verify/controls/aws_ec2_eip.rb

```diff
@@ -0,0 +1,37 @@
+aws_public_ip = attribute("aws_public_ip", default: "", description: "The Elastic IP address, or the carrier IP address.")
+aws_instance_id = attribute("aws_instance_id", default: "", description: "The ID of the instance the address is associated with, if any.")
+aws_allocation_id = attribute("aws_allocation_id", default: "", description: "The allocation ID for the address.")
+aws_association_id = attribute("aws_association_id", default: "", description: "The association ID for the address.")
+aws_domain = attribute("aws_domain", default: "vpc", description: "Indicates whether the address is for use in EC2-Classic (standard) or in a VPC (vpc).")
+aws_network_interface_id = attribute("aws_network_interface_id", default: "", description: "The ID of the network interface that the address is associated with, if any.")
+aws_network_interface_owner_id = attribute("aws_network_interface_owner_id", default: "", description: "The AWS account ID of the owner.")
+aws_private_ip_address = attribute("aws_private_ip_address", default: "", description: "The private IP address associated with the Elastic IP address.")
+aws_public_ipv_4_pool = attribute("aws_public_ipv_4_pool", default: "", description: "The Elastic IPV4 pool address.")
+aws_network_border_group = attribute("aws_network_border_group", default: "", description: "A unique set of Availability Zones, Local Zones, or Wavelength Zones from where AWS advertises IP addresses.")
```

Comment:
None of the values except `aws_instance_id` is present in `outputs.tf`.. please assign and test once!